devel/Arduino/Portfile

```diff
@@ -28,14 +29,23 @@ long_description \
 
 homepage            https://www.arduino.cc/
 
-checksums           rmd160  79434c36d66d117f775359d5b8b5033d13005200 \
-                    sha256  117da12bd853164b4ba74b6bc5b7a7289a84b973dae754513ebb61c76309e392 \
-                    size    38714163
+checksums           rmd160  f4ea5d35ee4fbb26a51656fe12a4749a595c00c2 \
+                    sha256  62374d7bc683dd9bbe46b55985d23f9a64371c8a5ffed2a3937e3edc59fbc6fb \
+                    size    38731429
 
 depends_lib         port:apache-ant
 
 use_configure       no
 
+# Arduino builds only for macOS 10.8 or greater, but
+# I'm not sure how to check platform, the following lines don't seem to work.
+#
+#platform darwin 12 {
+#    pre-fetch {
+#        ui_error "${name} requires Mac OS X 10.8 or greater."
+#    }
+#}
```

Comment:
Yes, that is not correct because it only checks for darwin 12. Other ports have this check. I'll post something if I find one.

Reply:
This link should work for you:

https://trac.macports.org/wiki/PortfileRecipes#compare-osx-darwin-version

Reply:
@mf2k Thanks, that was useful. I wrote some lines that seem to work.
But what's the correct way to abort the script?
```
pre-fetch {
    # Arduino requires macOS 10.8 (Mountain Lion) or greater (darwin 12)
    set check.os.major 12
    if {${os.major} < ${check.os.major}} {
        ui_error "${name} requires Mac OS X 10.8 or greater."
        # TODO HERE I NEED TO ABORT
    }
}
```

Reply:
to abort use something like

```
  return -code error "Unsupported OSX version"
```